Sources/Orbit/Foundation/Elevations/Elevation.swift

```diff
@@ -0,0 +1,97 @@
+import SwiftUI
+
+/// Use elevation to bring content closer to users.
+///
+/// Elevation levels with higher numbers are usually visually closer to the user.
+public enum Elevation {
+    case level1
+    case level2
+    case level3
+    case level4
+}
+
+public extension View {
+
+    /// Elevates the view to make it more prominent.
+    func elevation(_ level: Elevation, cornerRadius: CGFloat = 0) -> some View {
+        modifier(ElevationModifier(level: level, cornerRadius: cornerRadius))
+    }
+}
+
+struct ElevationModifier: ViewModifier {
+
+    let level: Elevation
+    let cornerRadius: CGFloat
+
+    let baseShadowColor = Color(red: 79 / 255, green: 94 / 255, blue: 113 / 255)
+    let borderWidth: CGFloat = 0.1
+
+    func body(content: Content) -> some View {
+        content
+            .clipShape(shape)
+            .overlay(shape.stroke(baseShadowColor.opacity(0.25), lineWidth: borderWidth))
+            .shadow(color: shadowColor, radius: shadowRadius, y: shadowOffsetY)
```

Comment:
Lvl2+ are hard to mimic with a single shadow. 

Maybe an optional use (parameter driven) of both `compositingGroup` + `drawingGroup` could achieve the desired look

https://developer.apple.com/documentation/swiftui/view/drawinggroup(opaque:colormode:)